Options.py

```diff
@@ -256,8 +256,10 @@ def from_text(cls, text: str) -> Range:
                     return cls(int(round(random.triangular(random_range[0], random_range[1], random_range[1]))))
                 else:
                     return cls(int(round(random.randint(random_range[0], random_range[1]))))
-            else:
+            elif text == "random":
                 return cls(random.randint(cls.range_start, cls.range_end))
+            else:
+                raise Exception(f"random text \"{text}\" did not resolve to a recognized pattern.")
```

Comment:
```suggestion
                raise Exception(f"random text \"{text}\" did not resolve to a recognized pattern. Acceptable values are: random, random-high, random-middle, random-low,  random-range-low-<min>-<max>, random-range-middle-<min>-<max>, random-range-high-<min>-<max>, or random-range-<min>-<max>.")
```

Since this exception eventually reaches a user, we should provide an exception message with instructions on remediation.

I know my suggestion is wordy, but it's informative. That is the best thing we can do for user facing exceptions.